libtelemetry/src/main/java/com/mapbox/android/telemetry/Attachment.java

```diff
@@ -0,0 +1,62 @@
+package com.mapbox.android.telemetry;
+
+import android.os.Parcel;
+import android.os.Parcelable;
+
+import com.google.gson.annotations.SerializedName;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Attachment extends Event implements Parcelable {
```

Comment:
I wonder if we could get to a more consistent naming conventions here? Attachment vs. VisionAttachment: should attachment event be more generic?